apps/web/src/pages/templates/workflow/digest/BackOffTooltipIcon.tsx

```diff
@@ -1,4 +1,89 @@
+import { useMantineTheme } from '@mantine/core';
+
 export const BackOffTooltipIcon = () => {
+  const { colorScheme } = useMantineTheme();
+
+  if (colorScheme === 'light') {
+    return (
+      <svg xmlns="http://www.w3.org/2000/svg" width="280" height="57" viewBox="0 0 280 57" fill="none">
```

Comment:
please run all the svg's through this tool: https://jakearchibald.github.io/svgomg/, to make them smaller in size

Reply:
nope, because it's svg's vector graphics :) it just removes unnecessary staff and improved the paths